examples/resources/hpegl_metal_host/resource.tf

```diff
@@ -25,11 +25,18 @@ resource "hpegl_metal_host" "terra_host" {
   image              = "ubuntu@18.04-20201102"
   machine_size       = "Medium System"
   ssh                = [hpegl_metal_ssh_key.newssh_1.id]
-  networks           = ["Public", "Storage-Client"]
+  networks           = ["Public", "Storage"]
   network_route      = "Public"
   location           = var.location
   description        = "Hello from Terraform"
   volume_attachments = [hpegl_metal_volume.iscsi_volume.id]
+  ## uncomment below to override the 60m timeouts
+  ## see https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/retries-and-customizable-timeouts
+  # timeouts {
```

Comment:
this example file gets used here https://registry.terraform.io/providers/HPE/hpegl/latest/docs/resources/metal_host#example-usage
So, no need to comment. It can be an example for using a timeout block.
Are the times for various actions enough? The default time for deletion is 60m, and in the example, too less.

Reply:
having this here is convenient for testing.  can you clarify "Are the times for various actions enough" ?

Reply:
These examples are used for official plugin documentation generation, So better to give a good example.
The timeout for deletion in the example is 5m, is this generally this short?

Reply:
I have not timed deletes, but it generally should be 5 min or less.   However, I expect users to override timeout values to increase values if needed to prevent failures.  I would not expect users would set this timeout low to cause failures.